scripts/compile-ajv-validators.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+
+const fs = require('fs')
+const path = require('path')
+const Ajv = require('ajv')
+const standaloneCode = require('ajv/dist/standalone').default
+const addFormats = require('ajv-formats')
+const schema = require('@uniswap/token-lists/dist/tokenlist.schema.json')
+
+const ajv = new Ajv({ code: { source: true, esm: true } })
+addFormats(ajv)
+const validate = ajv.compile(schema)
+let moduleCode = standaloneCode(ajv, validate)
+fs.writeFileSync(path.join(__dirname, '../src/utils/__generated__/validateTokenList.js'), moduleCode)
+
+const ajv2 = new Ajv({ code: { source: true, esm: true } })
+addFormats(ajv2)
+const validate2 = ajv2.compile(Object.assign({}, schema, { required: ['tokens']}))
```

Comment:
Why did you switch from a spread operator to `Object.assign` (which I think broadens typings, so is less strict)?

Reply:
temp workaround to linter not liking the spread operator ([related stack overflow](https://stackoverflow.com/questions/45120789/spread-operator-and-eslint)). I think we need to specify ecmaVersion to be >=2018 in the eslint-config package to fix that.